tests/testthat/test-add_data.R

```diff
@@ -88,3 +88,73 @@ test_that("addMarkov throws errors.", {
   expect_error(addMarkov(dd, transMat = mat6, chainLen = 5, wide = TRUE, start0lab = "xy"), class = "simstudy::start0probNotInTransMat")
   
 })
+
+# addSynthetic ----
+
+test_that("addSynthetic throws errors.", {
+  
+  ### Create fake "real" data set
+  
+  d <- defData(varname = "a", formula = 3, variance = 1, dist = "normal")
+  d <- defData(d, varname = "b", formula = 5, dist = "poisson")
+  d <- defData(d, varname = "c", formula = 0.3, dist = "binary")
+  d <- defData(d, varname = "d", formula = "a + b + 3*c", variance = 2, dist = "normal")
+  
+  A <- genData(1000, d, id = "index")
+  
+  def <- defData(varname = "x", formula = 0, variance = 5)
+  
+  S <- genData(120, def)
+  
+  expect_error(ddSynthetic(dtFrom = A))
```

Comment:
And this is why we check for error messages/error class :D there is a typo so this will always error but not with the error you expect 😁

Reply:
That was weird - not sure why I left those out.